pkg/ovsdb/transact.go

```diff
@@ -377,14 +378,20 @@ func NewCondition(condition []interface{}) (*Condition, error) {
 	if !ok {
 		return nil, errors.New(E_INTERNAL_ERROR)
 	}
+	schema := tableSchema.LookupColumn(column)
```

Comment:
`schema` name can be misleading, what about `columnSchema` or `cSchema` ...

Reply:
ok